otx/algorithms/classification/configs/base/data/semisl/data_pipeline.py

```diff
@@ -0,0 +1,68 @@
+"""Data Pipeline of Semi-SL model for Classification Task."""
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+# pylint: disable=invalid-name
+__img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+__resize_target_size = 224
+
+__common_pipeline = [
+    dict(type="Resize", size=__resize_target_size),
+    dict(type="RandomFlip", flip_prob=0.5, direction="horizontal"),
+    dict(type="AugMixAugment", config_str="augmix-m5-w3"),
+    dict(type="RandomRotate", p=0.35, angle=(-10, 10)),
+]
+
+__train_pipeline = [
+    *__common_pipeline,
+    dict(type="PILImageToNDArray", keys=["img"]),
+    dict(type="Normalize", **__img_norm_cfg),
+    dict(type="ImageToTensor", keys=["img"]),
+    dict(type="ToTensor", keys=["gt_label"]),
+    dict(type="Collect", keys=["img", "gt_label"]),
+]
+
```

Comment:
Maybe I think we can use below code (just suggestion).

For the `data_pipeline.py`.
```
__common_pipeline = [
    dict(type="Resize", size=__resize_target_size),
    dict(type="RandomFlip", flip_prob=0.5, direction="horizontal"),
    dict(type="AugMixAugment", config_str="augmix-m5-w3"),
    dict(type="RandomRotate", p=0.35, angle=(-10, 10)),
]
__unlabeled_strog_pipeline = [
    dict(type="MPARandAugment"),
    dict(type="PILImageToNDArray"),
    dict(type="ImageToTensor"),
    dict(type="Collect"),
]

__unlabeled_weak_pipeline = [
    dict(type="PILImageToNDArray"),
    dict(type="ImageToTensor"),
    dict(type="Collect"),
]
```

And then, we can use
```
img = common_pipeline(img)
strong_img = unlabeled_strong_pipeline(img)
weak_img = unlabeled_weak_pipeline(img)
```

Reply:
Reflected. Please review again